kod/object/active/holder/room/necarena.kod

```diff
@@ -626,7 +652,13 @@ messages:
 
    GetWatcher()
    {
-      return poWatcher;
+      local oWatcher;
+
+      % FindHoldingActive returns the first instance of Goad
```

Comment:
This comment is kind of redundant.  You could just change the whole function to:

return Send(self, @FindHoldingActive, #class=&Goad);

Reply:
Ok - I will make this change.  The fact that FindHoldingActive returns either the object or $ (instead of zero or False or any other return type) is important for Goad but it is well described in Holder so it isn't really needed here.